code/game/objects/items/weapons/RCL.dm

```diff
@@ -0,0 +1,152 @@
+/obj/item/weapon/twohanded/rcl
+	name = "rapid cable layer (RCL)"
+	desc = "A device used to rapidly deploy cables. It has screws on the side which can be removed to slide off the cables. Do not use without insulation!"
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "rcl-0"
+	item_state = "rcl-0"
+	opacity = 0
+	force = 5 //Plastic is soft
+	throwforce = 5
+	throw_speed = 1
+	throw_range = 7
+	w_class = WEIGHT_CLASS_NORMAL
+	origin_tech = "engineering=4;materials=2"
+	var/max_amount = 90
+	var/active = 0
+	var/obj/structure/cable/last = null
+	var/obj/item/stack/cable_coil/loaded = null
```

Comment:
Uhh, this is fairly standard practice across the rest of the codebase; is there any reason to not have the nulls? It's an excellent way of designating its status, instead of having future coders assume it.